docs/fsharp/language-reference/members/constructors.md

```diff
@@ -82,9 +82,11 @@ The following version of the previous code illustrates the combination of ordina
 
 [!code-fsharp[Main](../../../../samples/snippets/fsharp/lang-ref-2/snippet3507.fs)]
     
+## Constructors in inherited class
+You can definition constructors combined inherited class. For more information, see [Constructors and inheritance](../inheritance.md#constructors-and-inheritance).
```

Comment:
Not sure if "must" is appropriate here, since, if I understand it correctly, that only applies if the derived class has a primary constructor. If it has only additional constructors, then you still use `inherit`, but it's not directly in the derived class.

Maybe shorten that sentence to:

> When inheriting from a base class that has a constructor, you must specify its arguments in the `inherit` clause.

And let the link explain the details.